unitTests/rest/SearchEndpointTest.ts

```diff
@@ -639,6 +640,37 @@ export function SearchEndpointTest() {
           });
         });
 
+        it('for facetSearch', done => {
```

Comment:
This test is too big and should be broken up into multiple smaller ones.

As a rule of thumb, you want to have one `expect` per `it` block with a good test description. When a test fails, we will have a specific idea as to what broke whereas right now we will just know it is something related to FacetSearch.

Reply:
I don't want to have this test too different from the rest of them for the endpoint. It's often more the expect vs result than the description that gives me an idea as to what breaks usually.